pootle/apps/pootle_language/models.py

```diff
@@ -122,6 +122,28 @@ def direction(self):
 
     # # # # # # # # # # # # # #  Methods # # # # # # # # # # # # # # # # # # #
 
+    @classmethod
+    def get_canonical(cls, language_code):
+        """Returns the canonical `Language` object matching `language_code`.
+
+        The matching will be done using a regular expression. If no language can
```

Comment:
Oh, I now realize this was written based on the old implementation that was using a regex. I think we can simply remove the fist sentence here.